### PR TITLE
Resolve "ghe-restore with host arg and config value" test

### PR DESCRIPTION
**Pull request by DangMH**
`ghe-ssh "$GHE_HOSTNAME" -- 'ghe-config app.actions.enabled'` always returns exit code 0 even if the actions config is disabled.

The fix resolves that by evaluating the config value then doing the appropriate comparison.

### CODE REVIEW

File: test/testlib.sh [hunk 1]
```diff
@@ -525,7 +525,12 @@ enable_actions() {
 }
 
 is_actions_enabled() {
-  ghe-ssh "$GHE_HOSTNAME" -- 'ghe-config --true app.actions.enabled'
+  # ghe-ssh will always return exit code 0.  Evaluate property value instead.
+  if $( ghe-ssh "$GHE_HOSTNAME" -- 'ghe-config app.actions.enabled' ) = "true"; then
```
Comment (jianghao0718) [inline test/testlib.sh:529]: 'ghe-config app.actions.enabled' is possible to return non zero exit code if this is not set. Probably a combination of `--present` and `--true` would help here

**Review by AliMousai — APPROVED** [review]

**Comment by sonialey25** [comment]
> > - [ ] > > > []()[[___~~**_@@@@_**~~___]()]()

File: test/testlib.sh (hunk 1 above)
Comment (sonialey25) [reply test/testlib.sh:529]: there are hackers ising my info vell numbers and many account also be anle to change passwords  using Otiplex

**Comment by q464pp** [comment]
可以拉取合併嗎謝謝

**Comment by davidallen21151** [comment]
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by q464pp** [comment]
請問可以拉取合併嗎謝謝

**Pull request closed without merging**